### PR TITLE
Landing Map component completed & unit test on geolocation and fetch error

### PR DESCRIPTION
**Pull request by valerieyang00**
To Review:
- LandingMap / Test
- MapInfoWindow / Test

Map onLoad:
map onload

onClick Marker:
onclick marker

### CODE REVIEW

File: src/components/partials/LandingMap.tsx [hunk 1]
```diff
@@ -27,14 +28,21 @@ export default function LandingMap() {
     height: '60vh',
   }
 
+  const divStyle = {
```
Comment (moyara) [inline src/components/partials/LandingMap.tsx]: better name

File: src/components/partials/MapInfoWindow.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import { Restaurant } from '../../client/flavorite/models'
+
+type infoWindowProps = {
+  restaurants: Restaurant[]
+  coords: { lat: number; lng: number }
+}
+
+export default function MapInfoWindow({ restaurants, coords }: infoWindowProps) {
+  const displayInfoWindow = restaurants.map((restaurant, idx) => {
+    if (restaurant.latitude === coords.lat && restaurant.longitude === coords.lng) {
```
Comment (moyara) [inline src/components/partials/MapInfoWindow.tsx]: could be better to filter the restaurants with the coords first and map.
restaurants.filter(...).map(...)

**Review by moyara — APPROVED** [review]
looks good. minor nits but approve to unblock.

File: src/components/partials/LandingMap.tsx (hunk 1 above)
Comment (valerieyang00) [reply src/components/partials/LandingMap.tsx]: changed to 'infoBoxStyle'

File: src/components/partials/MapInfoWindow.tsx (hunk 2 above)
Comment (valerieyang00) [reply src/components/partials/MapInfoWindow.tsx]: yup thanks. Edited to:
```const displayInfoWindow = restaurants
    .filter(
      (restaurant) => restaurant.latitude === coords.lat && restaurant.longitude === coords.lng,
    )
    .map((restaurant, idx) => {...}```

**Pull request merged**